Finish intro sentence (#208)
mengelbart/rtp-over-quic-draft: draft-ietf-avtcore-rtp-over-quic.md

# Directions for Future Work {#futures}

> This document describes RoQ in sufficient detail that an implementer can build a RoQ application, but we recognize that additional work is likely, after we have sufficient experience with RoQ to guide that work. Possible directions would include future work resulting from experience ({{futures-impl-deploy}}) and work resulting from new QUIC extensions ({{futures-new-ext}}).

Reviewer: ```suggestion
This document describes RoQ in sufficient detail that an implementer can build a RoQ application, but we recognize that additional work is likely, after we have sufficient experience with RoQ to guide that work ({{futures-impl-deploy}}) and and as new QUIC extensions become available ({{futures-new-ext}}).
```
